bash/cloner.sh

```diff
@@ -25,26 +38,55 @@ make_folder () {
   echo
 }
 
-clone_repos () {
-  curl -s https://api.github.com/users/$account_name/repos | jq -c '.[]' |
+check () {
+  if [ -n "$1" ]
+    then # non-null/non-zero string check ... aka exists!
+      echo "success"
+      return 0; # return something similar to an exit code.
+    else
+      # echo "error"
+      return 1; # return something similar to an exit code.
```

Comment:
For later PR: Can we print why it failed? If the user doesn't exist the folder will be empty. If we hit an API rate limit some repos might be cloned but not all.